package.json

```diff
@@ -44,6 +45,9 @@
     "@storybook/addon-storyshots": "^6.0.26",
     "@storybook/addon-viewport": "^6.0.21",
     "@storybook/react": "^6.0.21",
+    "@types/node": "^14.14.17",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
```

Comment:
There may be a few more type packages we should add, for the other third party dependencies that we use.

Reply:
I'll need your help in careful evaluation of this.